### PR TITLE
/graph/tags fix lint errors/warnings

### PR DESCRIPTION
**Pull request by sharmaansh21**
Addresses #14756
Signed-off-by: Anshul Sharma optimisticanshul@gmail.com

### CODE REVIEW

**Comment by justanshulsharma** [comment]
@GordonTheTurtle fixed :bomb:

File: graph/tags/tags.go [hunk 1]
```diff
@@ -6,8 +6,10 @@ import (
 	"github.com/docker/distribution/registry/api/v2"
 )
 
+// Define DEFAULTTAG
```
Comment (calavera) [inline graph/tags/tags.go]: what about something like `DEFAULTTAG defines "latest" as the tag for images with implicit tag`

Comment (justanshulsharma) [inline graph/tags/tags.go]: Sounds good to me :+1:

**Comment by justanshulsharma** [comment]
@calavera done :+1:

**Comment by calavera** [comment]
I'm afraid that the tags package was cleaned up in https://github.com/docker/docker/commit/d4836cd7ec1c085c5a5caa7eb7f5eda4ace10eb6 and this is not necessary anymore. I'm sorry but I'm going to close this PR since those constants have been documented already.

**Pull request closed without merging**